src/common/http/httpc.hpp

```diff
@@ -1,16 +1,12 @@
 #pragma once
 
-#include "connection.hpp"
 #include <cstdint>
 #include <optional>
 #include <variant>
-#include <http/types.h>
-
-namespace connect_client {
+#include "connection.hpp"
+#include "http/types.h"
 
-enum class Method {
-    Post
-};
```

Comment:
The fact we can now reuse the same `http::Method` for both is good.

But I suspect there's some kind of translation of Method -> string somewhere and it now contains only the `Post`, while `Method` now has more variants. 

https://github.com/prusa3d/Prusa-Firmware-Buddy/blob/master/src/connect/httpc.cpp#L29

Also, that now probably belongs to types.cpp.